File: Data Structures & Algorithms/jump-game/submission-0.py
class Solution:
    def canJump(self, nums: List[int]) -> bool:
        req =1
        for i in range(len(nums)-2,-1,-1):
            if nums[i]>=req:
                req=1
            else:
                req+=1
            
        if req>=2:
            return False
        else:
            return True
            
        

File: Data Structures & Algorithms/single-number/submission-0.cpp
class Solution {
public:
    int singleNumber(vector<int>& nums)
    {
        int xor_p = 0;
        for(int i=0;i<nums.size();i++)
        {
            xor_p ^=nums[i];
        }
        return xor_p;
    }
};
